conda/conda-build/meta.yaml

```diff
@@ -177,13 +178,14 @@ requirements:
 {% endif %}
 {% if ucx_configured_bool %}
     - ucx >=1.14
+    - openmpi
 {% endif %}
 
   run_constrained:
     - __glibc >=2.17  # [linux]
     - python != 3.9.7
 {% if gpu_enabled_bool %}
-    - __cuda >={{ cuda_version }}
+    - __cuda
```

Comment:
Testing for a particular version of cuda does not seem to make sense as what we really need is a compatible driver, and there is no way to test for a driver. So, for now, we assume that it is enough to have any version of cuda, and if the driver is incompatible, we will get errors later.